static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -663,34 +668,36 @@ export class AccessMethod extends Component {
                       nested
                     >
                       {
-                        selectedVariables.length > 0 && (
-                          <p className="access-method__section-status">
-                            {`${selectedVariables.length} ${pluralize('variable', selectedVariables.length)} selected`}
-                          </p>
-                        )
-                      }
-
-                      {
-                        selectedVariables.length === 0 && (
+                        !hasVariables ? (
```

Comment:
nitpick; My preference would be to do 
```suggestion
                        hasVariables ? (
```
and just flip the order to make it less confusing but, I don't think its an issue